### PR TITLE
Added execution time to APRSummary

### PR DESCRIPTION
**Pull request by lisandrasilva**
Needed to implement [Issue 389](https://github.com/runtimeverification/kontrol/issues/389) in Kontrol.

### CODE REVIEW

File: src/pyk/proof/reachability.py [hunk 1]
```diff
@@ -760,6 +762,7 @@ def lines(self) -> list[str]:
             f'    terminal: {self.terminal}',
             f'    refuted: {self.refuted}',
             f'    bounded: {self.bounded}',
+            f'    execution time: {round(self.exec_time,2)}',
```
Comment (palinatolmach) [inline src/pyk/proof/reachability.py:765]:
```suggestion
            f'    execution time: {round(self.exec_time, 2)}',
```

Comment (palinatolmach) [reply src/pyk/proof/reachability.py:765]: If we're printing this information, shall we add `s` (for seconds) to the line? (or `ms`, depending on what we're measuring the time it)

I.e., something like
```
            f'    execution time: {round(self.exec_time, 2)}s',
```

Comment (lisandrasilva) [reply src/pyk/proof/reachability.py:765]: You are right! It's in `s`, but I am wondering if it wouldn't make sense to present the result in minutes.

Comment (lisandrasilva) [reply src/pyk/proof/reachability.py:765]: @palinatolmach what do you think of displaying it as `execution time: 0h 1m 33s`?

Comment (palinatolmach) [reply src/pyk/proof/reachability.py]: Looks great!

File: src/pyk/proof/reachability.py [hunk 2]
```diff
@@ -760,6 +762,7 @@ def lines(self) -> list[str]:
             f'    terminal: {self.terminal}',
             f'    refuted: {self.refuted}',
             f'    bounded: {self.bounded}',
+            f'    execution time: {self.exec_time//3600}h {(self.exec_time%3600)//60}m {self.exec_time%60}s',
```
Comment (palinatolmach) [inline src/pyk/proof/reachability.py:765]:
```suggestion
            f'    execution time: {self.exec_time // 3600}h {(self.exec_time % 3600) // 60}m {self.exec_time % 60}s',
```

**Review by palinatolmach — APPROVED** [review]

**Pull request merged**